util/log/context_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Tobias Schottdorf
+
+package log
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestContextLog(t *testing.T) {
+	ctx := Background().With(
+		Method, "Put",
+		NodeID, 5,
+		Err, fmt.Errorf("everything broke"))
+	expS := "NodeID=5 Method=Put Err=everything broke: %s is good"
```

Comment:
Shouldn't there be quotes around the values? Something like `Method="Put" Err="everything broke"`